house_interiors_1.logic

```diff
@@ -1,14 +1,73 @@
-area HouseInteriors1
-  room HearthBackDoor
-  room HearthBottom
-  room HearthCheap
-  room HearthDeluxe
-  room HearthGood
-  room HearthOracles
-  room HearthTop
-  room LibraryBottom
-  room LibraryTop
-  room Mayor
-  room PostOffice
-  room SchoolLeft
-  room SchoolRight
+area HouseInteriors1:
+  room HearthBackDoor:
+    node Main:
+      chest HP
+      door Door
+  room HearthBottom:
+    node Main:
+      door South
+      door Left
+      door Middle
+      door Right
+      door Stair
+  room HearthCheap:
+    node Main:
+      door South
+      chest Chest
+  room HearthDeluxe:
+    node Main:
+      door South
+      chest Chest
+  room HearthGood:
+    node Main:
+      door South
+      chest Chest
```

Comment:
I don't think these chests should be here - they aren't shuffleable at the moment, and if they would be they'd still not be accessible by coming in through the door. (Do we want to encode the other `exit`s into these rooms (from renting a room)? Probably?)